Formula/sourcery.rb

```diff
@@ -13,11 +13,10 @@ class Sourcery < Formula
   depends_on :xcode => ["8.0", :build]
 
   def install
-    ENV.delete("CC")
-    ENV["SDKROOT"] = MacOS.sdk_path
-    system "swift", "build", "-c", "release"
-    bin.install ".build/release/sourcery"
-    lib.install Dir[".build/release/*.dylib"]
```

Comment:
I tried to make it work without installing *.app bundle, but it will not work in some cases.
Sourcery has support for three types of templates, and two of them require some files bundled inside *.app bundle (https://github.com/krzysztofzablocki/Sourcery/blob/master/Sourcery/Generating/Template/JavaScript/JavaScriptTemplate.swift#L38 https://github.com/krzysztofzablocki/Sourcery/blob/master/Sourcery/Generating/Template/Swift/SwiftTemplate.swift#L123). Using this kind of templates will lead to a crash with unbundled binary (current version built with "swift build -c release" will crash too).
I think having an `.app` is a lesser evil than doesn't work in 2 of 3 cases